lib/stream.js

```diff
@@ -79,6 +79,10 @@ function createReadableStream({ url, fetch, options = {} }) {
         .pipeThrough(new EventSourceParserStream());
 
       for await (const event of streamAsyncIterator(stream)) {
+        if (event.event === "done") {
```

Comment:
This is an unrelated breaking change that I reverted, but we might want to circle back on this.